web/themes/custom/pineapple/components/_card.twig

```diff
@@ -1,9 +1,10 @@
+{{ attach_library('pineapple/card') }}
 {% set card_type = card_type|default('horizontal') %}
 {% set attributes = attributes|default(create_attribute()).addClass([
   'card-container',
   card_type,
 ]) %}
```

Comment:
`.setAttribute('data-card', '')` can be added after the ")" before the % sign on line 6